### ISSUE TITLE
Docs: Syntax Error in functions definition

### ISSUE THREAD

**Issue by taghikhavari** [microsoft/TypeScript-Website#465]
**Page URL:** 
https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html
**Issue:** 
There is some mistakes in function definition at the end of the page where we have:
```
// Takes two objects and merges them together
function extendextends {}, Second extends {}>(
  first: First,
  second: Second
): First & Second {...}
```
**Recommended Fix:** 
// Takes two objects and merges them together
function extend<First extends {}, Second extends {}>(
  first: First,
  second: Second
): First & Second {...}
**Browser:** 
all browser, chrome, brave, firefox

**Comment by taghikhavari** [comment]
Although I talk to @orta on twitter and he said maybe the problem is the code formatter: 

> "The code formatter is in-house and occasionally inperfect"

**Comment by orta** [comment]
I'd wager it is https://github.com/microsoft/TypeScript-Website/issues/283

**Comment by orta** [comment]
Confirmed this is fixed by #405 - http://www.staging-typescript.org/docs/handbook/unions-and-intersections.html#mixins-via-intersections
